### PR TITLE
Second parameter removed from NSLocalizedStrings.

### PR DESCRIPTION
**Pull request by ArkadiuszBanas**

### CODE REVIEW

File: BuyPolish/Pola.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1016,7 +1016,7 @@
 				GCC_TREAT_WARNINGS_AS_ERRORS = YES;
 				INFOPLIST_FILE = "Pola/Pola-Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
-				PRODUCT_BUNDLE_IDENTIFIER = "com.kj.$(PRODUCT_NAME:rfc1034identifier)";
+				PRODUCT_BUNDLE_IDENTIFIER = eu.ttmm.plebble.round;
```
Comment (paweljaneczek) [inline BuyPolish/Pola.xcodeproj/project.pbxproj]: ?

File: BuyPolish/Pola.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -987,7 +987,7 @@
...
```
Comment (paweljaneczek) [inline BuyPolish/Pola.xcodeproj/project.pbxproj]: ?

**Comment by paweljaneczek** [comment]
Reviewed :+1: Only two commenct about weird bundle identifier changes

**Comment by ArkadiuszBanas** [comment]
Damn it! This is PP I'm using to install app on device :)

**Pull request merged**